test/puppet_processor_test.rb

```diff
@@ -9,81 +9,87 @@ def test_deb
     input = File.read(File.join(File.dirname(__FILE__), "fixtures/puppet6-foreman-deb.yaml"))
     processor = Processor.new_processor("puppet", input)
     result = processor.build_report["host_report"]
+    body = JSON.parse(result["body"])
```

Comment:
Once implemented and some parameter like json_string=true is passed in, this is no longer needed.